### PR TITLE
Extra docs nanou

### PR DESCRIPTION
**Pull request by anaelChardan**
**Description (for Contributor and Core Developer)**

This PR adds the doc to reproduce a CI build by ourself.
It fixes a mistake on the jenkins file.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# How to reproduce a build on the CI?
+
+You have an error on the CI and you don't know why does it fail because it works on your computer?
```
Comment (tamarasaurus) [inline internal_doc/REPRODUCE_CI_BUILD.md]: ...and you don't know why it fails because..

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+You may want to reproduce the build!
+
+## Kubernetes
+
+First of all you have to setup the kubernetes client on your computer and gcloud on your computer
```
Comment (tamarasaurus) [inline internal_doc/REPRODUCE_CI_BUILD.md]: set up

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Install the cloud SDK: `https://cloud.google.com/sdk/docs/downloads-interactive`
+Login with your google akeneo account: `$ gcloud auth login`
+Configure gcloud with docker: `$ gcloud auth configure docker`
+Install the kubernetes client: `$ gcloud components install kubectl`
+Set the project (The one you want to reproduce the CI is named akecld-saas-training) : `$ gcloud config set project akecld-saas-training`
+Get the kubernetes credentials : `$ gcloud container clusters get-credentials europe-west3-a --zone=europe-west3-a`
+
+Here you should be able to do a `$ kubectl get all`
+
+Create a kubernetes namespace for you: `$ kubectl create namespace nanou for example`
```
Comment (tamarasaurus) [inline internal_doc/REPRODUCE_CI_BUILD.md]: yourself

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+## Continuous Integration
+
+Now your are setup you can reproduce your build!
```
Comment (tamarasaurus) [inline internal_doc/REPRODUCE_CI_BUILD.md]: now that you are set up

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+Let's take an example of this [Build](https://ci.akeneo.com/blue/organizations/jenkins/akeneo%2Fpim-enterprise-dev/detail/PR-4744/7/)
+
+You have the tab "Artefacts" at the top right corner, click on it.
+
+### Prepare the file
+
+Download the file ending by `_job.yaml`  which contains the the words selenium on it (because you will need behat most of the time) it should be the second file.
```
Comment (tamarasaurus) [inline internal_doc/REPRODUCE_CI_BUILD.md]: ending with
contains the word selenium

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+          fieldPath: metadata.name
+    - name: NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    volumeMounts:
+    - name: tmp-pod
+      mountPath: /tmp/pod
+```
+- Remove the nodeSelector and the tolerations at the end of the file, usually this block:
```
Comment (tamarasaurus) [inline internal_doc/REPRODUCE_CI_BUILD.md]: tolerations ?

File: internal_doc/REPRODUCE_CI_BUILD.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+To see if it is deployed you can type: `$ kubectl get pods -n nanou` to see if the containers are ready.
+
+Once they are ready you can go on the container to launch what you want by doing this command `$ kubectl exec -it jenkins-slave-abcd -c main bash`
+
+On it you can do the command that the CI launch they can be on the jenkinsfile or on the shared lib named "jenkins-k8s-utils" the user apache is `www-data` make sure you execute command with this one.
+
+### Destroy the cluster
+
+Now that you finished you can destroy your instance `$ kubect delete -f jenkins-slave-abcd.yaml -n nanou`
```
Comment (jjanvier) [inline internal_doc/REPRODUCE_CI_BUILD.md]: Now that you have finished you have to destroy your instance (don't forget that instances can be expensive!)  ...

**Comment by anaelChardan** [comment]
Thanks @tamarasaurus and @jjanvier ;)

**Pull request merged**